redaxo/src/addons/install/lib/command/download.php

```diff
@@ -18,7 +18,7 @@ protected function configure()
     {
         $this->setDescription('Download an AddOn from redaxo.org')
             ->addArgument('addonkey', InputArgument::REQUIRED, 'AddOn key, e.g. "yform"')
-            ->addArgument('version', InputArgument::OPTIONAL, 'Version, e.g. "3.2.1"');
+            ->addArgument('version', InputArgument::OPTIONAL, "Version, e.g. '3.2.1', '^3.2' or '3.*'");
```

Comment:
der kann jetzt auch das latest release mittels '*' ziehen. (z.B. `redaxo/bin/console install:download watson '*'`) :+1: 

Sollten wir für den User auch `latest` als alias zu `*` zulassen?
IMHO versteht jeder `latest` aber nicht zwangsweise version constraints

Reply:
Ich würde hier nix zulassen, was wir nicht auch in der package.yml bei den constraints können

Reply:
Auch glaube ich dass "latest" ohne definierte major version eine footgun ist